test/unit/driver/test_linode.py

```diff
@@ -0,0 +1,194 @@
+#  Copyright (c) 2018 Red Hat, Inc.
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to
+#  deal in the Software without restriction, including without limitation the
+#  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+#  sell copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+
+import os
+
+import pytest
+
+from molecule import config
+from molecule.driver import linode
+
+
+@pytest.fixture
+def _instance(patched_config_validate, config_instance):
+    return linode.Linode(config_instance)
+
+
+def test_config_private_member(_instance):
+    assert isinstance(_instance._config, config.Config)
+
+
+def test_testinfra_options_property(_instance):
+    assert {
+        'connection': 'ansible',
+        'ansible-inventory': _instance._config.provisioner.inventory_file
+    } == _instance.testinfra_options
+
+
+def test_name_property(_instance):
+    assert 'linode' == _instance.name
+
+
+def test_options_property(_instance):
+    x = {'managed': True}
```

Comment:
what is `x`? why not `a` or `b`? this name doesn't carry any context within. `expected_options`?